graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -111,6 +112,16 @@
 )
 
 
+# We prefer the explicit sys.version_info check instead of the more common try-except ImportError
+# approach, because at the moment mypy seems to have an easier time with the sys.version_info check:
+# https://github.com/python/mypy/issues/1393
+if sys.version_info[:2] >= (3, 8):
```

Comment:
Add this to the top-level `typedefs.py` file instead, so we don't have to do repeated conditional imports wherever we require `Protocol`.